src/components/tabs/tabs-theme.scss

```diff
@@ -97,25 +51,37 @@ md-tabs.md-THEME_NAME-theme {
     }
   }
 
+  &.md-accent, &.md-primary, &.md-warn{
+    > md-tabs-wrapper {
+      > md-tabs-canvas {
+        > md-pagination-wrapper {
+          > md-tab-item:not([disabled]) {
+            color: rgba(255, 255, 255, 0.7);
```

Comment:
I'm still thinking about using a hard-coded color. Is the Black Color also valid for dark themes? I know the specs doesn't mention that, I'm just curious. What do you think? :)